test/test.js

```diff
@@ -294,6 +294,45 @@ test('Remove mockjax definition by id', function() {
 	});
 });
 
+test('Remove mockjax definition by url', function() {
+	$.mockjax({
+		url: '/test',
+		contentType: 'text/plain',
+		responseText: 'test'
+	});
+
+	$.mockjax({
```

Comment:
Let's also add a test _without_ a default handler and ensure that we get an error callback.